data/tlg1271/tlg011/tlg1271.tlg011.1st1K-grc1.xml

```diff
@@ -82,10 +82,10 @@
         <monogr>
           <title xml:lang="lat">Clementinorum Epitomae Duae</title>
           <author xml:lang="lat">Clemens Romanus</author>
-          <editor> Albertus Rudolfus Maximilianus Dressel</editor>
+          <editor>Albertus Rudolfus Maximilianus Dressel</editor>
```

Comment:
This looks partially Latinized. Looks like WorldCat has Albert Rudolf Maximilian Dressel

Reply:
I will fix that, thanks for catching!